src/unstable_opts.rs

```diff
@@ -107,7 +107,7 @@ pub struct UnstableOptions {
     pub continue_on_errors: bool,
     pub paper_size: Option<String>,
     pub shell_escape: bool,
-    pub min_crossrefs: Option<i32>,
+    pub min_crossrefs: Option<u32>,
```

Comment:
I can revert this change if you want - I figured since they were 'unstable' it might be okay to swap out the type, and negative minimum values don't make sense anyways - they'd all just be equivalent to `0`

Reply:
Yeah, this is fine to change.